pkg/marathon/client.go

```diff
@@ -6,20 +6,35 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/http"
+	"net/url"
+	"os"
+	"regexp"
 	"strings"
 
 	"github.com/dcos/dcos-cli/api"
 	"github.com/dcos/dcos-cli/pkg/httpclient"
 	"github.com/dcos/dcos-core-cli/pkg/pluginutil"
-	"github.com/gambol99/go-marathon"
+	goMarathon "github.com/gambol99/go-marathon"
```

Comment:
Do we need this change?

Reply:
We agreed on renaming this import, so yes.